### PR TITLE
COMMERCE-11344 Visual issues for the Virtual Setting Override

### PR DESCRIPTION
**Pull request by pietrobocale**
https://issues.liferay.com/browse/COMMERCE-11344

Based from: #3698

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 1]
```diff
@@ -686,6 +686,210 @@ definition {
...
+		task ("And the admin logs in again") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Then now the Virtual Setting for the item are differents") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:869]:
```suggestion
		task ("Then the Virtual Setting for the item are different") {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:relevant

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**